lib/content-types/merge.js

```diff
@@ -155,6 +169,27 @@ const squish = (types) => {
         return plugin;
       });
 
+      // find attribute which is the identifier
+      const contender = attrs.find(attr => {
```

Comment:
Excellent. Didn't know `find` exists, can probably update `singleItem` with this